.github/workflows/tests-linux.yml

```
@@ -20,6 +20,8 @@ on:
       - "Formula/strace.rb"
 jobs:
   tests_linux:
+    needs: tap_syntax
+    if: github.event_name == 'pull_request' && needs.tap_syntax.outputs.run-tests
```

```suggestion
    if: needs.tap_syntax.outputs.run-tests
```

It looks like this workflow is only run on `pull_request` events anyway, so no need to check for it.